duo_client/client.py

```diff
@@ -337,6 +337,21 @@ def json_api_call(self, method, path, params):
         (response, data) = self.api_call(method, path, params)
         return self.parse_json_response(response, data)
 
+    def json_paging_api_call(self, method, path, params):
+        objects = []
+        next_offset = 0
+
+        if 'limit' not in params and self.paging_limit:
+            params['limit'] = str(self.paging_limit)
+
+        while next_offset is not None:
+            params['offset'] = str(next_offset)
+            (response, data) = self.api_call(method, path, params)
+            (objects, metadata) = self.parse_json_response(response, data)
```

Comment:
You are going to need to add a v2 of parse_json_response. The version that's in this edition of the client doesn't return a tuple with both the response data and the metadata. It currently just returns the response data.